bindata/network/multus/multus.yaml

```
@@ -39,6 +39,8 @@ data:
     # detect which version we're using in order to copy the proper binaries
     case "${ID}" in
       rhcos|scos)
+        IFS=':' read -ra parts <<< "$CPE_NAME"
```

probably a non-issue but should this be
```suggestion
        local IFS=':' read -ra parts <<< "$CPE_NAME"
```
so the changed IFS doesn't leak?

---

Quoting @trozet:

> probably, and should probably be in its own function

Let's address this
/hold

---

hmm, I didn't think it would leak because its being set inline, but may IFS is special, either way I could just use
> echo $CPE_NAME | cut -f 4 -d :
like the lines below

---

I think you may be right about the inline setting of IFS here.

Nonetheless, I personally find using `cut` easier to understand. Mind changing it?

---

Done, I'd have used cut in the first place if I noticed the pattern already being used in the file...